src/index.js

```diff
@@ -477,6 +480,19 @@ class Enmap extends Map {
     this.changedCB = cb;
   }
 
+  /**
+   * Shuts down the database. WARNING: USING THIS MAKES THE ENMAP UNUSABLE. You should
+   * only use this method if you are closing your entire application.
+   * This is already done by Enmap automatically on shutdown unless you disabled it.
+   * @returns {Enmap} The enmap.
+   */
+  close() {
+    this[_readyCheck]();
+    instances.splice(instances.indexOf(this), 1);
```

Comment:
Instances will only contain the closing enmap if autoclosure is enabled so this should check to make sure it's there before splicing